### ISSUE TITLE
Pretrained Networks missing some layer values

### ISSUE THREAD

**Issue by cfunk1210** [TengdaHan/DPC#10]
While loading the pretrained networks for fine-tuning, it cannot find the running mean and variance for the batch normalization layers.  Why are these not saved in the model that was exported? track_running_stats is true in resnet_2d3d? 

```
Weights not loaded into new model:
module.backbone.bn1.running_mean
module.backbone.bn1.running_var
module.backbone.bn1.num_batches_tracked
module.backbone.layer1.0.bn1.running_mean
module.backbone.layer1.0.bn1.running_var
module.backbone.layer1.0.bn1.num_batches_tracked
module.backbone.layer1.0.bn2.running_mean
module.backbone.layer1.0.bn2.running_var
module.backbone.layer1.0.bn2.num_batches_tracked
module.backbone.layer1.1.bn1.running_mean
module.backbone.layer1.1.bn1.running_var
module.backbone.layer1.1.bn1.num_batches_tracked
module.backbone.layer1.1.bn2.running_mean
module.backbone.layer1.1.bn2.running_var
module.backbone.layer1.1.bn2.num_batches_tracked
module.backbone.layer2.0.bn1.running_mean
```

**Comment by TengdaHan** [comment]
We choose to not save the running mean and variance when training DPC for 2 reasons: 1. the final downstream task is always on different datasets, therefore keeping the running stat of pre-training dataset is meaningless. 2. save some memory

**Comment by cfunk1210** [comment]
Fair enough, just making sure that was accurate.
